docs/05-staking/01-what_is_staking.md

```diff
@@ -0,0 +1,9 @@
+---
+title: What is staking
+---
+
+## What is staking?
```

Comment:
Make this just one `#` so that it is a `<h1>` and it doesn't render again on the page